services/user-svc/main.go

```diff
@@ -114,6 +123,15 @@ func (userServiceServer) CreateUser(ctx context.Context, request *api.CreateUser
 		log.Info().Str("userID", userID).Msg("created new user")
 	}
 
+	// prepare for event
+	user.ID = &userID
+	user.Credentials = nil
+
+	// publish USER_CREATED event
+	if err := daprClient.PublishEvent(ctx, "pubsub", "USER_CREATED", user); err != nil {
+		log.Error().Err(err).Msg("could not publish USER_CREATED event")
```

Comment:
It is less likely that publishing the event will fail instead of the keycloak operation. But we have the same issue as before. When the creation of the event fails, the user was created, the organization gets not created. Okay for now.
`TODO: I guess the Transactional Outbox Pattern is a good use for that.`

Reply:
As I said, okay for now. I've added the comment.